server/models/Cuisine.py

```diff
@@ -0,0 +1,40 @@
+from . import db
+from marshmallow import fields, Schema
+
+
+# Join table linking users and their favorite cuisines
+favorite_cuisines_table = db.Table('favorite_cuisines',
+                                   db.Column('user_id', db.Integer,
+                                             db.ForeignKey('users.id'), primary_key=True),
+                                   db.Column('cuisine_id', db.Integer,
+                                             db.ForeignKey('cuisines.id'), primary_key=True)
+                                   )
```

Comment:
Not a big deal but I would name this something like users_cuisines. Just more obvious that it is a join table.

Reply:
good point.I think we should also keep favorite_cuisines in the title because thats what the table data represents. how about a table name of `favorite_cuisines(users_cuisines)`?

Reply:
just changed table name to `favorite_cuisines(users_cuisines)`